Userland/Applications/Spreadsheet/SpreadsheetWidget.cpp

```diff
@@ -55,7 +55,10 @@ SpreadsheetWidget::SpreadsheetWidget(GUI::Window& parent_window, Vector<NonnullR
             auto docs = sheet_ptr->gather_documentation();
             auto help_window = HelpWindow::the(window());
             help_window->set_docs(move(docs));
-            help_window->set_window_mode(GUI::WindowMode::Modeless);
+
+            if (!help_window->is_visible())
+                help_window->set_window_mode(GUI::WindowMode::Modeless);
```

Comment:
Shouldn't this be set at initialization time rather than every time we show the window?